docs/config/microshift-starter.ks

```diff
@@ -0,0 +1,72 @@
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+text
+reboot
+
+# Configure network to use DHCP and activate on boot
+network --bootproto=dhcp --device=link --activate --onboot=on --hostname=microshift-starter.local --noipv6
```

Comment:
@ggiguash why do we disable IPv6, do we have any issue related to that?

Reply:
@mangelajo - MicroShift is not using/supporting it at the moment. I've been doing it in the devenv. Just to be on the safe side.

Reply:
We haven't tested the IPv6 features, but having ipv6 on the host would not hurt, anyway, better to be on the safe side I guess 👍